apps/workers/deduper/worker.go

```diff
@@ -75,16 +76,8 @@ func newWorker(info string) task.Worker {
 		return task.InvalidWorker(err.Error())
```

Comment:
I know this wasn't you, but it might be better to return the error in the way it was intended? 
```go
	if err != nil {
		return task.InvalidWorker("error: %s", err.Error()))
	}
```

Reply:
I can change it, but the task will already have an error result in it. Isn't it redundant to also include an `error:` in the message field?

Reply:
it is a bit redundant, maybe more like `invalid worker: %s`